sound/soc/sof/intel/hda-dai.c

```diff
@@ -546,7 +510,6 @@ struct snd_soc_dai_driver skl_dai[] = {
 },
 {
 	.name = "SSP5 Pin",
-	.ops = &ssp_dai_ops,
```

Comment:
this has got nothing to do with multi-core
Split your work into logical parts please and focus ONLY on multi-core support in this PR

Reply:
> this has got nothing to do with multi-core
> Split your work into logical parts please and focus ONLY on multi-core support in this PR

This is not needed for the dynamic pipeline.

So the dependency here is: we need to make the dynamic pipeline work for all platforms on Core 0 first, then we should add these fixes, then enable multi-core on top of it.

That's why I have to do invasive change in the series here.